### PR TITLE
Enhanced Export to Excel with occupant information SPC-41 #Resolve

### PR DESCRIPTION
**Pull request by cfscosta**

### CODE REVIEW

File: src/main/java/org/fenixedu/spaces/domain/occupation/Occupation.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ public boolean overlaps(Interval... intervals) {
     }
 
     public Boolean isActive() {
-        return !getIntervals().get(getIntervals().size() - 1).getEnd().isBeforeNow();
+        return getIntervals().stream().anyMatch(interval -> interval.contains(DateTime.now()));
```
Comment (pedrosan7os) [inline src/main/java/org/fenixedu/spaces/domain/occupation/Occupation.java]: Is this change related to the issue?

File: src/main/java/org/fenixedu/spaces/domain/occupation/SharedOccupation.java [hunk 2]
```diff
@@ -97,7 +97,7 @@ public boolean overlaps(Interval... intervals) {
 
     @Override
     public Boolean isActive() {
-        return !getIntervals().get(getIntervals().size() - 1).getEnd().isBeforeNow();
+        return getIntervals().stream().anyMatch(interval -> interval.contains(DateTime.now()));
```
Comment (pedrosan7os) [inline src/main/java/org/fenixedu/spaces/domain/occupation/SharedOccupation.java]: Is this change related to the issue?

Comment (cfscosta) [inline src/main/java/org/fenixedu/spaces/domain/occupation/SharedOccupation.java]: sort of... the old isActive was wrong... means to an end
